app/assets/javascripts/custom.js

```diff
@@ -39,12 +56,25 @@ $(document).ready(function(){
     $("#cart").html('<i class="material-icons left">shopping_cart</i>' + _totalItemsInCart);
   }
 
+  var line_prep_total = function(qty, prep_time) {
+    var added_time = parseInt(qty/7) * 10;
+    var time = added_time + prep_time;
+    return time
+  }
+
+
   $(".qty-editable-width").change(function(){
     var food_id = $(this).data('message');
     var qty = parseFloat($(this).val());
     var price = parseFloat($('#food_sub_total_' + food_id).data('message'));
+    var prep_time = parseFloat($('#food_prep_' + food_id).data('message'));
     var line_total = parseFloat(qty * price).toFixed(2);
+    $('#food_sub_total_' + food_id).text(line_total);
```

Comment:
Identifier 'food_id' is not in camel case.<br>Identifier 'line_total' is not in camel case.